### DISCUSSION TITLE
When will v2.4.2 of xUnit be released?

### DISCUSSION THREAD

**Discussion by innix** [xunit/xunit#2526] [Question] [answered]
Version `2.4.2-pre.12` of xUnit was published to NuGet over 7 months ago and [includes support](https://github.com/xunit/xunit/discussions/2414#discussioncomment-1600003) for static analysis of NRT's[^1], which was introduced in C# 8. This version of C# was released in September 2019, that's over 2 and a half years ago, so it's being widely used by a lot of projects by now. I'm a bit surprised that this feature still isn't being supported by a stable release of xUnit. Are there any plans to release a finalized version of `2.4.2` soon?

[^1]: NRT = nullable reference types. See [Microsoft docs](https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references).

**Comment by jscott-concord** [comment]
Great question. I had a developer that joined our organization recently saying we should be using Nunit as it is being maintained and xunit.net has not had a release since 2015. On the main page, the last release shows 2.1 but I can see a release 2.4.1 back in 2018 so I'm guessing that was an official release, and documentation is just not updated?

I would love to see things get officially reflected and updated so they do not have a foot to stand on. ;-)

**Reply by MartyIX** [reply]
I like this summary regarding xUnit: https://github.com/xunit/xunit/discussions/2437. It does not answer your question probably but it's worth reading I think.

**Comment by bradwilson** [comment, marked as answer]
I just pushed 2.4.2-pre.27 to NuGet, and my plan is to mark that as the final build for 2.4.2 unless anybody comes up with any showstoppers. Please try it out. https://www.nuget.org/packages/xunit/2.4.2-pre.27

**Reply by innix** [reply]
Looks like the final build of `v2.4.2` has been released on NuGet: https://www.nuget.org/packages/xunit/2.4.2

I'll mark this as answered.
